app/controllers/users_controller.rb

```diff
@@ -0,0 +1,63 @@
+class UsersController < ApplicationController
```

Comment:
Hi Anna, you actually do not need a users controller for these assignments. Devise takes care of all of this for us. It takes care of the routing, controllers, and views. Doing it manually can cause issues if it tries to interfere with the Devise controllers and views.